docs/commands/unbind.md

```diff
@@ -14,13 +14,14 @@ Deletes a service binding.
 
 |Optional|Global Flag|
 |--------|-----------|
-| -h, --help  Help for unbind command.| No |
-| -f, --force Force delete without confirmation | No |
-| --id ID of the service binding. Required when name is ambiguous| No |
-| --mode  How calls to Service Manager are performed sync or async (default "async")| No |
-| -o, --output Output format of the command. Possible opitons: json, yaml, text.| No|
-| --config Set the path for the smctl config.json file (default is $HOME/.sm/config.json).|Yes|
-| -v, --verbose Use verbose mode.|Yes|
+| -h, --help  Help for unbind command. | No |
+| -f, --force Use this parameter to delete a resource without raising a confirmation message. | No |
+| --force-delete Delete the service binding and all of its associated resources from the database. Use this parameter if service instance cannot be properly deleted. This parameters can only be used by operators with technical access. | No |
+| --id ID of the service binding. Required when name is ambiguous. | No |
+| --mode  How calls to Service Manager are performed sync or async (default "async"). | No |
```

Comment:
Whether to use synchronous or asynchronous calls to Service Management.
The default value is 'async'.